ci/publish-crate.sh

```diff
@@ -21,7 +21,7 @@ CRATES=(
   drone
   instruction-processors/{budget_api,config_api,storage_api,token_api,vote_api}
   runtime
-  instruction-processors/{budget,bpf_loader,config,vote,storage,token,vote}
+  instruction-processors/bpf_loader
```

Comment:
The removed programs are still needed, just need to add a `-api` suffix to each of them in most cases.

eg. wallet needs `vote-api`: https://github.com/solana-labs/solana/blob/master/wallet/Cargo.toml#L25

I suspect wallet will also need `budget_program` too: https://github.com/solana-labs/solana/blob/master/wallet/Cargo.toml#L29.

To catch all these, I'd checkout the v0.13.0 tag and then try to run `ci/publish-crate.sh` manually with the right env vars.  Then adjust the `CRATES` array as needed to get the publish to work to identify the missing bits.  With the missing bits known, then update `ci/publish-crate.sh` with the result.